README.md

```diff
@@ -26,26 +26,6 @@ LOCATION 's3://osm-pds/planet/';
 ```
 
 ```sql
-CREATE EXTERNAL TABLE planet_history (
```

Comment:
This should stay in the README as there's still a need to have both versions of the dump available.